### ISSUE TITLE
Basic authentification (user+password)

### ISSUE THREAD

**Issue by oddmean** [tinyproxy/tinyproxy#158]
Blame me a newb but I can't still understand:
Is Tinyproxy capable of handling user-password authentication? I've red the man, but I still have no clue.

**Comment by rofl0r** [comment]
yes, it is in current master. see https://github.com/tinyproxy/tinyproxy/pull/129 for details.

**Comment by graynk** [comment]
Does it support multiple users or just one?

**Comment by rofl0r** [comment]
multiple. you could have just tried it out before asking.

**Comment by rofl0r** [comment]
that should be possible, and wouldn't be very hard to implement.
i mean the idea using an external script, whereas adding built-in support for LDAP etc is entirely out of scope.
PRs welcome, and you might want to open a new issue for your idea regardless, so it doesn't get lost.
